### PR TITLE
Blk 1648 update documentation

### PR DESCRIPTION
**Pull request by bpbirch**
Please read through the updated documentation and make sure everything looks good.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,16 +36,17 @@ Do you love SIMBA Chain? Do you love Hardhat? Then you're in luck! The Hardhat p
...
 2. cd into that directory and start an npm project.
-3. install hardhat in that project / directory. This directory, where your package.json will live, is where you will run your Hardhat commands from.
-4. install the SIMBA Chain Hardhat plugin
-5. add a 'require' or 'import' statement to your hardhat.config.js or hardhat.config.ts file, depending on which file your project has.
-6. create a simba.json file in the top level of your project, and populate that file with 'baseURL' and 'web3Suite' fields.
-7. run `npx hardhat simba help` to make sure the plugin is installed
+3. Install hardhat in that project / directory. This directory (the root of your project, where your package.json will live), is where you will run your Hardhat commands from.
+4. Install the SIMBA Chain Hardhat plugin
+5. Add a 'require' or 'import' statement to your hardhat.config.js or hardhat.config.ts file, depending on which file your project has.
+6. (Optional) create a simba.json file in the top level of your project, and populate that file with a 'baseURL' field.
```
Comment (JehoG) [inline README.md:44]: 'C'reate

File: README.md [hunk 2]
```diff
@@ -35,16 +36,17 @@ Do you love SIMBA Chain? Do you love Hardhat? Then you're in luck! The Hardhat p
...
+7. (Necessary if using client credentials) Create an env file (.simbachain.env, simbachain.env, or .env) in your project root, and specify client credentials as well as your SIMBA Chain base API URL.
+8. Run `npx hardhat simba help` to make sure the plugin is installed
 
 ## Prerequisites
-You should have a SIMBA Enterprise Platform Instance to communicate with. Additionally you must have a least one contract application created in the instance. To create an application, open your browser, navigate to your instance and log in using your SIMBA user account. Click on your organization -> Applications and then click on the "Add" button. Follow the on screen instructions to create your application.
+You should have a SIMBA Blocks Instance to communicate with. Additionally you must have a least one contract application created in the instance (though the plugins will direct you to create one if you do not yet have one). To create an application, open your browser, navigate to your instance and log in using your SIMBA user account. Click on your organization -> Applications and then click on the "Add" button. Follow the on screen instructions to create your application.
```
Comment (JehoG) [inline README.md:49]: though the plugin's' ? Does it take a 's' there?

File: README.md [hunk 3]
```diff
@@ -124,14 +126,21 @@ require("./simba_hardhat");
 
 ## Project Settings
 
-To use the SIMBA Chain Hardhat plugin, you will need create and configure a simba.json file. Your simba.json file should live in the top level of your Hardhat project, where your package.json file lives, and should contain values for baseURL and web3Suite:
+The most important file in your SIMBA Chain Hardhat project is your simba.json file. You DO NOT need to create and configure a simba.json file. However, the one thing that does absolutely have to be configured is a value for SIMBA_API_BASE_URL, which you can configure in either your simba.json file, or in your project's env file (.simbachain.env, simbachain.env, or .env).
 
-NOTE: the following baseURL is an example, and will likely be different for your environment
+If you want to configure SIMBA_API_BASE_URL (you can also use the key of baseURL instead) in your simba.json, you will need to create and configure a simba.json file. Your simba.json file should live in the root of your Hardhat project, where your package.json file lives, and should contain values for SIMBA_API_BASE_URL (or baseURL):
```
Comment (JehoG) [inline README.md:131]: should contain value's'?

File: README.md (hunk 2 above)
Comment (bpbirch) [reply README.md:49]: I was referring to the hardhat and truffle plugins. I'll change to plugin

File: README.md (hunk 3 above)
Comment (bpbirch) [reply README.md:131]: I think I already fixed this one

**Review by JehoG — APPROVED** [review]

**Pull request merged**